firestore/client.go

```diff
@@ -148,11 +148,12 @@ func (c *Client) Doc(path string) *DocumentRef {
 	return doc
 }
 
+// TODO(crwilcox): Commented out, may be best to not extend surface at client level.
```

Comment:
Nice, I think removing this is good. Best not to add additional ways of doing things without a clear rationale.